### ISSUE TITLE
Painless Mass

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10078]
**Preferred term label:**
Painless Mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C1711205

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Breast fibroadenoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
new modifier term

HP:0025761
Painless mass
def: This modifier is applied to a palpable mass or swelling in the body that does not elicit pain upon palpation.
